### DISCUSSION TITLE
Toxic repos

### DISCUSSION THREAD

**Discussion by stravnik** [toxic-repos/toxic-repos#35] [General]
Do you have any suggestions?
Write to us about them.

**Comment by mykola-drone-dev** [comment]
Delete this

**Reply by stravnik** [reply]
Why?

**Reply by mykola-drone-dev** [reply]
To those russians that are 'outsode of politics': ru.wikipedia.org/wiki/Репортаж_с_петлёй_на_шее
To @stravnik - if there's a need to explain, there's already no need to explain :)

**Reply by stravnik** [reply]
We got a great dialogue (no)

**Reply by mykola-drone-dev** [reply]
There's nothing to talk about.

**Reply by webknjaz** [reply]
How about because by nature, **_this repository is a political manifestation of the terrorist nazi government_** in its ukrainophobic pursuit of genocide and ethnic cleansing as demonstrated by numerous actions and claims in the README?

For example, this is copied from the readme:
> UTF characters with the image of the Ukrainian flag

**Reply by stravnik** [reply]
Are you really stubborn?
If the flag of Russia was used for sabotage, then UTF characters with the image of the Russian flag would be written there.

**Reply by webknjaz** [reply]
Stubborn or not, I'm just staring the obvious facts. This repo is spreading driven by the fascist ideology (by definition). Nothing more, nothing less. I've seen people here repeating the ruscist propaganda, without any modifications. Including your posts. It's also a simple fact, it is what it is.
Given the above, it's simply incompatible to state that the project is not political while the actions demonstrate the opposite — glorifying the support of the terrorist state, in particular.

**Reply by igorsaux** [reply]
Есть suggestion добавить Code of Conduct и сразу блокировать подобных выше индивидов. Вести конструктивный и цивилизованный диалог они все равно не умеют, только дискредитируют себя и все сообщество открытого кода.

**Reply by stravnik** [reply]
@igorsaux Есть хорошие примеры Code of Conduct?

**Reply by igorsaux** [reply]
@stravnik да [вот](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md) гитхабовский вполне подойдёт. Много же не надо - я считаю достаточно взять пункты из секции `Our Standards`.

Причём уже беглым взглядом можно заметить нарушение некоторых пунктов из этого CoC:
- `Being respectful of differing opinions, viewpoints, and experiences`
- `Giving and gracefully accepting constructive feedback`
- `Focusing on what is best not just for us as individuals, but for the overall community`
- `Trolling, insulting or derogatory comments, and personal or political attacks`
- `Other conduct which could reasonably be considered inappropriate in a professional setting`

К слову, у некоторых владельцев или мейнтенеров репозиториев из списка токсичных есть те же "стандарты поведения", которые, видимо, в их случаях двойные.
